include/vapor/RenderParams.h

```diff
@@ -484,46 +497,89 @@ class PARAMS_API RenderParams : public ParamsBase {
     static const string LightingEnabledTag;
 
-    //! If a renderer supports rotation about a point of origin,
-    //! this string identifies the parameter for the origin's
+    //! If a renderer supports rotation about a point of origin (IE - Slice and Contour),
+    //! this tag identifies the parameter for the origin's
     //! location on the X axis.
+    //! Applies to data of type: double
+    //! Valid values: A point within the data domain's X axis coordinates
     static const string XSlicePlaneOriginTag;
 
-    //! If a renderer supports rotation about a point of origin,
-    //! this string identifies the parameter for the origin's
+    //! If a renderer supports rotation about a point of origin (IE - Slice and Contour),
+    //! this tag identifies the parameter for the origin's
     //! location on the Y axis.
-
+    //! Applies to data of type: double
+    //! Valid values: A point within the data domain's Y axis coordinates
     static const string YSlicePlaneOriginTag;
-    //! If a renderer supports rotation about a point of origin,
-    //! this string identifies the parameter for the origin's
+
+    //! If a renderer supports rotation about a point of origin (IE - Slice and Contour),
+    //! this tag identifies the parameter for the origin's
     //! location on the Z axis.
+    //! Applies to data of type: double
+    //! Valid values: A point within the data domain's Z axis coordinates
     static const string ZSlicePlaneOriginTag;
 
-    //! If a renderer supports rotation about a point of origin,
-    //! this string identifies the parameter for the rotation
+    //! If a renderer supports rotation about a point of origin (IE - Slice and Contour),
+    //! this tag identifies the parameter for the rotation
     //! about the X axis.
+    //! Applies to data of type: double
+    //! Valid values: -90.0 to 90.0
     static const string XSlicePlaneRotationTag;
 
-    //! If a renderer supports rotation about a point of origin,
-    //! this string identifies the parameter for the rotation
+    //! If a renderer supports rotation about a point of origin (IE - Slice and Contour),
+    //! this tag identifies the parameter for the rotation
     //! about the Y axis.
+    //! Applies to data of type: double
+    //! Valid values: -90.0 to 90.0
     static const string YSlicePlaneRotationTag;
 
-    //! If a renderer supports rotation about a point of origin,
-    //! this string identifies the parameter for the rotation
+    //! If a renderer supports rotation about a point of origin (IE - Slice and Contour),
+    //! this tag identifies the parameter for the rotation
     //! about the Z axis.
+    //! Applies to data of type: double
+    //! Valid values: -90.0 to 90.0
     static const string ZSlicePlaneRotationTag;
 
-    //! If a renderer samples data points along a vector,
-    //! this string identifies the parameter for how many samples
+    //! If a renderer samples data points along a vector (IE - Slice and Contour),
```

Comment:
Does sampling apply to the contour renderer? I thought that it operated on the native grid with an appropriate marching algo in all cases?

Reply:
The sampling rate applies to the *deep breath* ArbitrarilyOrientedRegularGrid class, so I believe the contour renderer uses it too.  @Stasj?

Reply:
That is correct.

Reply:
Maybe the caveat here is that this only applies when 3D data are sliced (for contouring or pseudo coloring)

Reply:
Ok, I've added the caveat you've outlined to the slice/contour tags with \details and \copydetails.